nodejs/awsx/examples/api/index.ts

```diff
@@ -86,4 +87,32 @@ const api = new awsx.apigateway.API("myapi", {
     requestValidator: "ALL",
 });
 
+/**
+ * Export the url of the API.
+ */
 export const url = api.url;
+
+/**
+ * Set up the Usage Plan for the API.
+ */
+const usagePlan = new aws.apigateway.UsagePlan("my-usage-plan", {
+    apiStages: [{
+        apiId: api.restAPI.id,
+        stage: api.stage.stageName,
+    }],
+});
+
+/**
+ * Create an API Key for testing purposes.
+ */
+const apikey = new aws.apigateway.ApiKey("my-key", {});
+const usagePlanKey = new aws.apigateway.UsagePlanKey("my-key", {
+    keyId: apikey.id,
+    keyType: "API_KEY",
+    usagePlanId: usagePlan.id,
+});
```

Comment:
what's the general relation between apikeys, plankeys, and plans?   are there 1:1 relations for any of them?  1:n?  n:m?

Reply:
- many usage plans per many APIs (in a usage plan you define things like rate & throttling limits & this can be applied across many APIs & stages)
- many apikeys per usage plan (this is typically 1 apikey per client)
- 1 plankey per apikey (this is what associates and apikey to a usage plan)

Reply:
ok great.  that will help inform how we can structure helpers here.  for example, since there are "many apikeys per usage plan",  we'd likely want `usagePlan.createApiKey`